src/SOS/SOS.UnitTests/ConfigFiles/Windows/Debugger.Tests.Config.txt

```diff
@@ -181,6 +209,9 @@
         <Option>
           <TestName>SOS.StackAndOtherTests</TestName>
         </Option>
+        <Option>
+          <TestName>DotnetDumpCommands</TestName>
+        </Option>
```

Comment:
Why did you add this?   This and the above block of options for DotnetDumpCommands should be needed.

Reply:
Same as my previous answer

Reply:
Yes, you are right if your test has a TestName, then the config filtering will remove all the configs that don't have the TestName property.  It has been too long since I've been in this code.  

You might be able to add this instead to the ` Default (prebuilt)` section to get the same effect. 

```
      <Options>
        <Option>
        </Option>
        <Option>
          <TestName>DotnetDumpCommands</TestName>
        </Option>
      </Options>
```

Reply:
1. If I add your portion of config just below the prebuilt configurations, it's not working because the code to parse confg file only looks for `Option` tag as children of `Options`, so what I just added is simply ignored 

```
<Options>
   <!--
    Default (prebuilt)
    -->
  <Option Condition="'$(RuntimeVersionLatest)' != ''"> <!-- Set the 3 variables to latest version --></Option>
  <Option Condition="'$(RuntimeVersion50)' != ''"><!-- Set the 3 variables to net50 --></Option>
  <Option Condition="'$(RuntimeVersion31)' != ''"><!-- Set the 3 variables to net31 version --></Option>
  <Option Condition="'$(RuntimeVersion21)' != ''"><!-- Set the 3 variables to net21 version --></Option>
        **<Options>
          <Option>
          </Option>
          <Option>
            <TestName>DotnetDumpCommands</TestName>
          </Option>
        </Options>**
        .....
</Options>
```

2. If I add the portion of config  after the `Options` tag containing the prebuilt configurations, I'm gonna duplicate every existing configuration with a `TestName` set to DotnetDumpCommands
```
<Options>
   <!--
    Default (prebuilt)
    -->
  <Option Condition="'$(RuntimeVersionLatest)' != ''"> <!-- Set the 3 variables to latest version --></Option>
  <Option Condition="'$(RuntimeVersion50)' != ''"><!-- Set the 3 variables to net50 --></Option>
  <Option Condition="'$(RuntimeVersion31)' != ''"><!-- Set the 3 variables to net31 version --></Option>
  <Option Condition="'$(RuntimeVersion21)' != ''"><!-- Set the 3 variables to net21 version --></Option>
  .....
</Options>
**<Options>
  <Option>
  </Option>
  <Option>
    <TestName>DotnetDumpCommands</TestName>
  </Option>
</Options>**
```

Reply:
In conclusion, it seems I can't easily improve this part, what do you think ?

Reply:
I'm ok with leaving this the way it is. Thanks for trying those options.